Centaurus.DAL/DiffObject.cs

```diff
@@ -1,15 +1,46 @@
 ﻿using Centaurus.DAL.Models;
+using Centaurus.Models;
 using MongoDB.Bson;
 using System.Collections.Generic;
 using System.Diagnostics;
 
 namespace Centaurus.DAL
 {
-    public class DiffObject
+    public class QuantumItem
     {
-        public List<QuantumModel> Quanta { get; set; } = new List<QuantumModel>();
+        public QuantumItem(QuantumModel quantum)
+        {
+            Quantum = quantum;
+            Effects = new Dictionary<int, EffectsModel>();
+        }
+
+        public QuantumModel Quantum { get; }
 
-        public List<EffectModel> Effects { get; set; } = new List<EffectModel>();
+        public Dictionary<int, EffectsModel> Effects { get; }
+
+        public int EffectsCount { get; private set; }
```

Comment:
```suggestion
        public int EffectsCount => effects.Effects.Count;
```

Reply:
Effects property is a dictionary that contains effects grouped by account. So ```Effects.Count``` will give us a number of accounts that were involved in a quantum processing.